lib/tasks/gettext.rake

```diff
@@ -29,6 +29,18 @@ namespace :gettext do
     clean_dir('locale_alaveteli_pro')
   end
 
+  Rake::Task['find'].clear
```

Comment:
Why doesn't the `gettext_i18n_rails`-provided task work?

Reply:
It does work but it will:
1. leave `*.edit.po` and `*.time_stamp.po` files in the locale directory
2. can't easily be re-used for the pro/theme find tasks

Reply:
👍